packages/react/src/components/ComposedModal/ModalFooter.tsx

```diff
@@ -156,6 +246,7 @@ ModalFooter.propTypes = {
   /**
    * The `ref` callback for the primary button.
    */
+  // @ts-expect-error: Invalid derived type
   inputref: PropTypes.oneOfType([
```

Comment:
I couldn't figure this one out, is there anything we can do here to fix the type error @tay1orjones @mbarrer ?
```
  /**
   * The `ref` callback for the primary button.
   */
  // @ts-expect-error: Invalid derived type
  inputref: PropTypes.oneOfType([
    PropTypes.func,
    PropTypes.shape({
      current: PropTypes.any,
    }),
  ]),

```
<img width="840" alt="image" src="https://user-images.githubusercontent.com/40550942/233740496-8d463cf4-6c8d-4813-b44f-93d513053863.png">